docs/site/Application-setup-tips.md

```diff
@@ -0,0 +1,80 @@
+---
+lang: en
+title: 'Tips for Application Setup'
+keywords:
+  LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Application Setup,
+  RestApplication
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Application-setup-tips.html
+---
+
+Here are some tips for application setup to help avoid common pitfalls and
+mistakes.
+
+### Extend from `RestApplication` when using `RestServer`
+
+If you want to use `RestServer` from the `@loopback/rest` package, we recommend
+extending `RestApplication` in your app instead of manually binding `RestServer`
+or `RestComponent`. `RestApplication` already uses `RestComponent` and makes
+useful functions in `RestServer` like `handler()` available at the app level.
+This means you can call the `RestServer` functions to perform all of your
+server-level setups in the app constructor without having to explicitly retrieve
+an instance of your server.
+
+### Serve static files
+
+The `RestServer` allows static files to be served. It can be set up by calling
+the `static()` API.
+
+```ts
+app.static('/html', rootDirForHtml);
+```
+
+or
+
+```ts
+server.static(['/html', '/public'], rootDirForHtml);
+```
+
+Static assets are not allowed to be mounted on `/` to avoid performance penalty
+as `/` matches all paths and incurs file system access for each HTTP request.
+
+The static() API delegates to
+[serve-static](https://expressjs.com/en/resources/middleware/serve-static.html)
+to serve static files. Please see
+https://expressjs.com/en/starter/static-files.html and
+https://expressjs.com/en/4x/api.html#express.static for details.
+
+**WARNING**:
```

Comment:
Actually we can remove this line, because we have the "warning note". It will be highlighted.